recipes-bsp/bootfiles/rpi-config_git.bb

```diff
@@ -118,10 +118,6 @@ do_deploy() {
 
     # Video camera support
     if [ "${VIDEO_CAMERA}" = "1" ]; then
-        # TODO: It has been observed that Raspberry Pi 4B 4GB may fail to enable the camera if "start_x=1" is at the end
-        #       of the file. The underlying cause is unknown, but it can be related with a file size limitation affecting
-        #       this variable. Therefore, "start_x=1" has been set to replace the original occurrence in config.txt,
```

Comment:
these lines are more than 80 columns and this could be a problem in config parser to handle that see 

https://github.com/raspberrypi/firmware/issues/1012#issuecomment-401030535

Please reformat these comments so that they fit into < 80 chars on line in final config ( e.g. line 122 is 112 characters in length) 

see if that fixes the issue you have.

Reply:
@SCVready any update on this?

Reply:
Oh sorry. Yes, I can update the comment and fit it in 80 chars columns. I'll update this pull request this weekend.